mypy/test/helpers.py

```diff
@@ -494,3 +495,11 @@ def normalize_file_output(content: List[str], current_abs_path: str) -> List[str
     result = [re.sub(r'\b' + re.escape(base_version) + r'\b', '$VERSION', x) for x in result]
     result = [timestamp_regex.sub('$TIMESTAMP', x) for x in result]
     return result
+
+
+def find_test_files(pattern: str, exclude: Optional[List[str]] = None) -> List[str]:
+    return [
+        str(x.name)
```

Comment:
`x.name` is already a string, you can remove the conversion.

Reply:
Also maybe use a bit more descriptive name than `x` (`path` maybe?)

Reply:
@evemorgen Could you please implement these two suggestions? ^^ I will then merge this.

Reply:
@ilevkivskyi Done in [7974ae6](https://github.com/python/mypy/pull/13143/commits/7974ae67cf3ca261d585c9aba5032b620274dadc) 🙂